source/extensions/transport_sockets/tls/ssl_socket.h

```diff
@@ -85,6 +86,7 @@ class SslSocket : public Network::TransportSocket,
   uint64_t bytes_to_retry_{};
   mutable std::string cached_sha_256_peer_certificate_digest_;
   mutable std::string cached_url_encoded_pem_encoded_peer_certificate_;
+  mutable std::string cached_pem_encoded_peer_cert_chain_;
```

Comment:
Nit: `cached_url_encoded_pem_encoded_peer_cert_chain_ `

Reply:
Addressed in 876e4a5.